components/kyma-environment-broker/internal/process/provisioning/initialisation.go

```diff
@@ -256,6 +249,24 @@ func (s *InitialisationStep) launchPostActions(operation internal.ProvisioningOp
 	return s.operationManager.OperationSucceeded(operation, msg)
 }
 
+func (s *InitialisationStep) createExternalEval(operation internal.ProvisioningOperation, instance *internal.Instance, log logrus.FieldLogger) (internal.ProvisioningOperation, time.Duration, error) {
+	pp, err := operation.GetProvisioningParameters()
+	if err != nil {
+		log.Errorf("cannot fetch provisioning parameters from operation: %s", err)
+		return s.operationManager.OperationFailed(operation, "invalid operation provisioning parameters")
+	}
+	if pp.PlanID == broker.TrialPlanID {
+		log.Info("skipping AVS external evaluation creation for trial plan")
+		return operation, 0, nil
+	}
+	log.Infof("creating external evaluation for instance %", instance.InstanceID)
+	operation, repeat, err := s.externalEvalCreator.createEval(operation, instance.DashboardURL, log)
+	if err != nil || repeat != 0 {
```

Comment:
nit: maybe it'll be better to log error message if it's not null and just `return operation, retry, nil`